data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,542 @@ mission "Heliarchs Buy Jump Drive Mission"
+	`	"I was originally born left-handed, but as a child I chose to learn how to use my right hand, because I thought being able to use both hands would be cool."`
+	label continue
+	`	The signing continues for several more hours. Eventually, the same Kimek in the bookstore uniform announces that the store is closing soon.`
+	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna walks with you back to your ship to say goodbye.`
+	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 100 credits daily, a small but significant portion of my royalties."`
+	`	Anna then takes out another copy of "Beyond the Hyperdrive", which she signs in front of you. It is the Coalition edition of the book, but translated to your language. Quickly reading through it, you realize that the book has been significantly altered from the version you read in human space. "Here, take it. A special copy just for you, as a thank you for all you've done."`
+	choice
+		`	"Why is this book so different from the one in human space?"`
+		`	"Did you make any changes to it?"`
+	`	"Well, for one, it's not in our language. Some concepts just don't translate very well, so I tried to capture the spirit of the story. The Heliarch linguists were very helpful," she explains. "They really went into depth on some aspects of the Coalition's languages. None of the major plot points were changed, thankfully, though some content in the book had to be edited or removed. My Heliarch friend said that if I changed this or removed that, the book would sell so much better, and as they said, they wanted to 'make it more suitable for the Coalition'. Their words.`
+	`	"As for the special copy, it was so I could verify and look at the changes they made, in my language, so I wouldn't have to learn their language. They agreed to print another copy for me so I could give it to you.`
+	`	"I've got to go now. I want to see how well my book is selling. Goodbye!"`
+	`	Anna then leaves your ship. You ponder what to do next, but you do feel rewarded having helped a struggling author make it big.`
+
+
+event "beyond the hyperdrive payment"
+
+mission "Beyond the Hyperdrive Gratitude Payment"
+	landing
+	source
+		government "Coalition" "Heliarch"
+	to offer
+		has "event: beyond the hyperdrive payment"
+	on offer
+		payment 2000000
+		conversation
+			`You are shocked to see an extra <payment> in your account. Checking your messages you open a message from Anna, the author whose book 'Beyond the Hyperdrive' was published in Coalition space a few months ago. She explains that the book has been selling very well in Coalition space and sends an expression of gratitude. Given the <payment> you received this is likely an understatement.`
```

Comment:
"Checking your messages you open a message" is a little awkward. Maybe "checking your communicator/phone/portable 31st century fax machine that's less advanced than current human technology" instead?